src/app/child-dev-project/notes/model/note.ts

```diff
@@ -56,7 +68,10 @@ export class Note extends Entity {
   }
 
   public getColorForId(entityId: string) {
-    if (this.category.isMeeting && !this.isPresent(entityId)) {
+    if (
+      this.category.isMeeting &&
+      this.childrenAttendance.get(entityId)?.status === AttendanceStatus.ABSENT
```

Comment:
I would either call the `childrenAttendace` `entityAttendance` or rename `entityId` to `childId`. The usage of both makes it confusing, whether this class is entity independent or just usable with the child class.